src/elements/input/input.svelte

```diff
@@ -225,13 +226,14 @@ const handleNumberDragDown = async (event: PointerEvent) => {
   </div>
   <input
     type={inputType}
+    {autocomplete}
     {placeholder}
     {name}
     {value}
     inputmode={isNumeric ? 'numeric' : undefined}
     pattern={inputPattern}
-    readonly={isReadonly || isDisabled}
-    aria-disabled={isDisabled}
+    readonly={isReadonly}
+    disabled={isDisabled}
```

Comment:
Same as above. For example for the fragment work we are doing, we want to use readonly inputs that are disabled. With this change fragment configs would be unnavigable and essentially hidden from users with assistive tech entirely. So they would have no idea what a fragments config values are.

Reply:
For this one, why not use `<v-input readonly>`?

Reply:
That should be fine, but we need to support `aria-readonly` for checkbox inputs IIRC

Reply:
> The value of `<input type="checkbox">` can not be edited making readonly not relevant. However, when creating checkboxes with role="checkbox" the aria-readonly attribute is supported.